### PR TITLE
feat: sssLimitReached modal and ErrorHandler

### PR DESCRIPTION
**Pull request by ragozin-nikita**
Fixes #

## Proposed Changes

- HQM-266
-
-

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

**Review by iGroza — APPROVED** [review]

File: src/helpers/wrappers/getMetadataValue.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {getMetadataValue} from '@haqq/shared-react-native';
```
Comment (iGroza) [inline src/helpers/wrappers/getMetadataValue.ts:1]: snake case naming

File: src/models/errorHandler.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import {makeAutoObservable} from 'mobx';
```
Comment (iGroza) [inline src/models/errorHandler.ts:1]: snake case naming

**Review by iGroza — CHANGES_REQUESTED** [review]

File: src/helpers/wrappers/getMetadataValue.ts (hunk 1 above)
Comment (ragozin-nikita) [reply src/helpers/wrappers/getMetadataValue.ts:1]: This is 3rd party dependency

File: src/models/errorHandler.ts (hunk 2 above)
Comment (ragozin-nikita) [reply src/models/errorHandler.ts:1]: This is 3rd party dependency

**Review by iGroza — APPROVED** [review]

**Pull request merged**